frontend/src/metabase/visualizations/click-actions/modes/EmbeddingSdkMode.ts

```diff
@@ -0,0 +1,28 @@
+import type { QueryClickActionsMode } from "../../types";
+import { DashboardClickAction } from "../actions/DashboardClickAction";
+import { HideColumnAction } from "../actions/HideColumnAction";
+import { NativeQueryClickFallback } from "../actions/NativeQueryClickFallback";
+
+export const EmbeddingSdkMode: QueryClickActionsMode = {
+  name: "embedding-sdk",
+  hasDrills: true,
+  availableOnlyDrills: [
```

Comment:
does `availableOnlyDrills` mean "available drills"?
```suggestion
  availableDrills: [
```